PwshSpectreConsole.Tests/formatting/new_Format-SpectreTable.tests.ps1

```diff
@@ -9,6 +9,10 @@ Describe "Format-SpectreTable" {
             $testBorder = 'Markdown'
             $testColor = Get-RandomColor
             Mock Write-AnsiConsole {
+                param(

```

Comment:
maybe try with param block,
```ps1
                param(
                    $RenderableObject
                )
```

Reply:
Haha wow that pester test was annoying. It turns out the github action runner uses PowerShell 7.2 which doesn't have a parameter set that allows `-Skip` + `-SkipLast` in the same command. The github runners are just powershell on linux but I didn't think they'd be using the n-1 LTS version

Reply:
They're using 7.2 https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/select-object?view=powershell-7.2

It looks like it has another test bug where the console width in the runner is set pretty small so the table column header being truncated with an ellipsis isn't being validated correctly.